packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -633,6 +633,18 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         setCenter(center);
     }, [center]);
 
+    /*
+     * Fallback mechanism for setting the center position.
+     * If no center is specified (neither through URL parameter nor prop),
```

Comment:
I don't think we should talk about URL parameters here. Any URL parameter is "transformed" to a prop in MapsIndoorsMap and passed on if the supportsUrlParameters prop is set to true. So talking about them here seems a little "late" 😄